Dependencies/xr/CMakeLists.txt

```diff
@@ -7,6 +7,7 @@ set(SOURCES "Include/XR.h"
 
 if (ANDROID)
     set(SOURCES ${SOURCES}
+        "Source/ARCore/Include/IXrContextARCore.h"
```

Comment:
Since this is already in the ARCore folder it seems like it should just be named IXrContext. That would be more similar to the rest of the code.

Reply:
I'm assuming this is a header we'll publish in which case I'd argue we should keep the names distinct so that a consuming module can be built to target multiple platforms (OpenXR, ARCore, etc.). While we could require that they end up with an include path that's like `#include <ARCore/IXrContext.h>` or something like that, I think keeping the files named separately is still good.

Reply:
talked with @ryantrem offline. If we want to support oculus quest we could have both IXrContextARCore.h and IXrContextOpenXR.h distributed with android. This naming choice is primarily to avoid file renames/breaking changes in the future if different platforms support multiple xr backends.